code-challenges.test.js

```diff
@@ -47,3 +126,31 @@ const testArray2 = [7, 8, 2, 3, 1, 5, 4]
 
 
 // b) Create the function that makes the test pass.
+
+
+
+
+// I am going to make a test for a function called removeDuplicates; it expect the function to take in two arrays and return one array with no duplicates
+
+describe ("removeDuplicates", () => {
+    it(" takes in two arrays and returns one with no duplicates", () => {
+        expect(removeDuplicates(testArray1, testArray2)).toEqual([3, 7, 10, 5, 4, 8, 2, 1])
+    })
+})
+
+// I got the function not defined error, now i will create the function, it will use one method to join the both arrays into one and then either use a method to remove duplicates or use .filter to remove any duplicates
+
+// I looked it up and you can use a set operator to automatically remove all duplicates, then you convert the value back into an array
```

Comment:
This is what I am looking for in your pseudocode.  I see all the steps taken in your code provided here.